packages/vscode-extension/media/walkthroughs/get-started/chatWithGitHubCopilot.md

```diff
@@ -9,4 +9,4 @@ To use GitHub Copilot Chat with `@teamsapp`, you must complete the following ste
 2. **[Install GitHub Copilot Chat for Visual Studio Code](command:fx-extension.installCopilotChat?%5B%22TeamsAgentWalkthrough%22%5D)**
 3. **[Install GitHub Copilot extension for Microsoft 365 Agents Toolkit](command:fx-extension.openInstallTeamsAgent?%5B%22TeamsAgentWalkthrough%22%5D)**
 
-After completing these steps, you can use `@teamsapp` to chat with GitHub Copilot and explore Teams app or Microsoft 365 Copilot agent development.
+After completing these steps, you can use `@m365agents` to chat with GitHub Copilot and explore Teams app or Microsoft 365 Copilot agent development.
```

Comment:
minor: This string may need further rewrite for "Teams app" later.

Reply:
Thanks for letting me know. Will track this.